index.html

```diff
@@ -116,8 +120,8 @@
                     <a class="list-group-item" href="https://mozilla.github.io/iacomus/resources/public/index.html#?config=https://s3-us-west-2.amazonaws.com/telemetry-public-analysis/mainthreadio/data/iacomus.json">
                         Main Thread I/O <div class="dashboard-description">View time taken by I/O operations on the main thread.</div>
                     </a>
-                    <a class="list-group-item" href="distribution">
```

Comment:
If you are replacing distribution with active-weekly, shouldn't you remove distribution from the tree?